collectors/Makefile.am

```diff
@@ -3,30 +3,41 @@
 MAINTAINERCLEANFILES = $(srcdir)/Makefile.in
 
 SUBDIRS = \
-    plugins.d \
-    apps.plugin \
-    cgroups.plugin \
-    charts.d.plugin \
-    checks.plugin \
-    cups.plugin \
-    diskspace.plugin \
-    fping.plugin \
-    ioping.plugin \
-    freebsd.plugin \
-    freeipmi.plugin \
-    idlejitter.plugin \
-    macos.plugin \
-    nfacct.plugin \
-    xenstat.plugin \
-    perf.plugin \
-    node.d.plugin \
-    proc.plugin \
-    python.d.plugin \
-    slabinfo.plugin \
-    statsd.plugin \
-    tc.plugin \
-   	$(NULL)
+	plugins.d \
+	apps.plugin \
+	cgroups.plugin \
+	charts.d.plugin \
+	checks.plugin \
+	cups.plugin \
+	diskspace.plugin \
+	fping.plugin \
+	ioping.plugin \
+	freebsd.plugin \
+	freeipmi.plugin \
+	idlejitter.plugin \
+	macos.plugin \
+	nfacct.plugin \
+	xenstat.plugin \
+	perf.plugin \
+	node.d.plugin \
+	proc.plugin \
+	python.d.plugin \
+	slabinfo.plugin \
+	statsd.plugin \
+	tc.plugin \
+	$(NULL)
+
+usercustompluginsconfigdir=$(configdir)/custom-plugins.d
+usergoconfigdir=$(configdir)/go.d
+
+# Explicitly install directories to avoid permission issues due to umask
+install-exec-local:
+	$(INSTALL) -d $(DESTDIR)$(usercustompluginsconfigdir)
+	$(INSTALL) -d $(DESTDIR)$(usergoconfigdir)
 
+uninstall-local:
```

Comment:
This looks different to the other cases. There isn't a special case for uninstalling the other subdirectories of the $[configdir)  (e.g. `charts.d` or `python.d`). What is the motivation for doing these two differently?

Reply:
Due to no .keep files for this directory, I though that `make uninstall` would not remove the directory. It turns out that `make uninstall` does not remove **any** directory.

I have now removed these `uninstall-local` recipes for consistency. There were only to be used by `make uninstall` anyway, which is not used (completely unrelated to netdata-uninstaller.sh).